internal/pkg/api/server/buffer/dependencies/store.go

```diff
@@ -40,6 +40,10 @@ func ExportsPrefix(projectID int, receiverID string) string {
 	return fmt.Sprintf("config/export/%d/%s/", projectID, receiverID)
 }
 
+func MappingsPrefix(projectID int, receiverID string, exportID string) string {
+	return fmt.Sprintf("config/mapping/revision/%d/%s/%s/", projectID, receiverID, exportID)
```

Comment:
Tu konkretne sa to neriesi (a u `projectId` to nema zmysel), ale radsej napisem:
- etcd zoradzuje keys lexikograficky
- https://tools.knowledgewalls.com/online-lexicographic-ascending-or-descending-sort
- preto, ak je v kluci nejake cislo, a definuje `order`
- ako bude tu za `/`, `revisionId`, a bude nas zaujimat posledna revizia
- tak key sa musi zarovnat s lava s `0`.
- vid obrazky (inac nam nebude fungovat ziskanie najnovsej revizie)

![image](https://user-images.githubusercontent.com/19371734/203015663-4ce7e36c-1440-4f41-af22-19f91791058f.png)

![image](https://user-images.githubusercontent.com/19371734/203015850-9860bc14-b776-4163-ab55-a4754aed800d.png)

Reply:
Ajaj, kua. Jasně, fixnu.